AbstractClient.php

```diff
@@ -91,10 +118,20 @@ protected function performRequest(Endpoint\EndpointInterface $endpoint)
         return $body;
     }
 
+    protected function getUriFromEndpoint(EndpointInterface $endpoint): string
+    {
+        $uri = $endpoint->uri();
+        if ($this->prependPath !== null) {
+            return $this->prependPath . $uri;
```

Comment:
Couldn't we just add this to the server url? https://swagger.io/docs/specification/api-host-and-base-path/
In this case we change the base uri from our guzzle client?

Reply:
No, Guzzle is following the RFC standard for relative URI's: https://docs.guzzlephp.org/en/stable/quickstart.html#creating-a-client.
TLDR: sdk's have their paths with a slash, so all paths after the domain are ignored in the base uri.